test/elm/comparison/data.coffee

```diff
@@ -3256,8 +3466,22 @@ define AEqB_Quantity_incompatible: 5 'Cel' != 5 'm'
 define ALtB_Quantity_incompatible: 5 'Cel' != 40 'm'
 ###
 
+###
+Translation Error(s):
+[12:40, 12:95] Could not resolve call to operator Equal with signature (tuple{a:System.String,b:System.Any},tuple{a:System.String,c:System.Any}).
```

Comment:
Have you looked into this error? Is this a valid test case that points out an issue in CQL-to-ELM? Or is it an invalid test case? If the latter, should it be fixed or removed?